commands/new_function.go

```diff
@@ -44,15 +44,23 @@ language or type in --list for a list of languages available.`,
 }
 
 func runNewFunction(cmd *cobra.Command, args []string) {
```

Comment:
We're changing the commands to return (error) so we can handle the exit codes during CI / scripting. If that's beyond scope of the PR please raise an issue instead to log the work.

Reply:
I will take you up on the second option: https://github.com/openfaas/faas-cli/issues/210